docs/Hierarchy-Storage-on-Tachyon.md

```diff
@@ -46,6 +46,10 @@ directories. If the quota for some storage directories are not set, the last quo
 There is default quota(128MB) for storage layer with alias MEM, if the quota for any other storage
 layer is not set, the system will report the error and exit the initialization.
 
+    $ tachyon.worker.data.folder
+Unlike the original configuration which is the absolute path of data directory, it becomes the
+relative path where data is stored in each storage directory.
+
```

Comment:
It should be here: https://github.com/amplab/tachyon/blob/master/docs/Configuration-Settings.md , what do you think?